bin/javascript-typescript-langserver

```diff
@@ -1,4 +1,22 @@
-#!/bin/bash
```

Comment:
I disagree with changing `javascript-typescript-langserver` and `javascript-typescript-stdio` from shell scripts to node scripts (other `bin` files are ok to be changed). My reasons are:
- we should be able to run our application on Windows as a part of editor plugin (see Atom, Sublime Text). I think that the best way to do it on Windows is `add bin to path, point to javascript-typescript-stdio in your plugin configuration`,  rather than instruct users to run `node ...\javascript-typescript-stdio`
- you cannot simply start server by running `javascript-typescript-stdio` on Windows, there should be a bat/cmd/exe file
- if you still think that we should use Node scripts, then let's at least drop TS duplicates from `src` directory